pkg/handlers/host.go

```diff
@@ -107,16 +109,33 @@ func (h *HostHandlers) PatchHostByID(w http.ResponseWriter, req *http.Request) e
 	}
 	hostToDB, err := h.constructHostForDB(createHostRequest, req)
 	if err != nil {
-		return err
+		slog.Error("Failed to constructHostForDB", slog.Any("error", err))
+		return api.WriteJSON(w, http.StatusInternalServerError, api.APIError{Error: err.Error()})
 	}
 	hostToDB.ID = id
+	hostGet, err := h.hostService.GetHostByID(hostToDB.ID)
```

Comment:
`hostGet` is now declared but not used. It still makes sense to check for host existence in DB, so this block of logic should be changed to:

```suggestion
	_, err := h.hostService.GetHostByID(hostToDB.ID)
```